Linux_basics_srinath

```diff
@@ -0,0 +1,141 @@
+pwd command is used to know the current directory.
+srinath@srinath-VirtualBox:~$ pwd
```

Comment:
I like the way you implemented every command. It would have been much better, if you have used indentation for the implementation part. If you have used indentation between command explanation and implementation. But overall I like the way you did.